frontend/common/src/liquity/index.ts

```diff
@@ -35,8 +35,8 @@ export type LiquityStakingDetailProxyEntries = z.infer<
 >;
 
 export const LiquityStakingDetail = z.object({
-  balances: LiquityStakingDetailEntry,
-  proxies: LiquityStakingDetailProxyEntries
+  balances: LiquityStakingDetailEntry.nullish(),
+  proxies: LiquityStakingDetailProxyEntries.nullish()
```

Comment:
They should be `nullable` instead of `nullish` since they will either have a value or be null they won't be undefined.